src/app/rpmostree-builtin-status.c

```diff
@@ -76,6 +76,25 @@ get_active_txn (RPMOSTreeSysroot *sysroot_proxy)
   return NULL;
 }
 
+static void
+print_packages (const char *k, guint max_key_len, const char *const* pkgs)
+{
+  g_autofree char *packages_joined = NULL;
+  g_autoptr(GPtrArray) packages_sorted =
+    g_ptr_array_new_with_free_func (g_free);
+  for (char **iter = (char**) pkgs; iter && *iter; iter++)
+    {
+      if (strchr (*iter, ' ') != NULL)
+        g_ptr_array_add (packages_sorted, g_strdup_printf ("'%s'", *iter));
```

Comment:
Ahh thanks, I knew there was a glib function for this! I tweaked it to be even more strict, since spaces are not the only possible chars we might encounter that require quoting.